packages/web-service/src/pages/method-statement/check-method-statement.js

```diff
@@ -1,7 +1,11 @@
 import { FILE_UPLOADS, TASKLIST } from '../../uris.js'
 import pageRoute from '../../routes/page-route.js'
-import { s3FileUpload } from '../../services/s3-upload.js'
-import { FILETYPES } from '../common/file-upload/file-upload.js'
+import Joi from 'joi'
+import { SECTION_TASKS } from '../tasklist/licence-type-map.js'
+import { APIRequests } from '../../services/api-requests.js'
+import { timestampFormatter } from '../common/common.js'
+
+const anotherFileUpload = 'another-file-check'
 
 export const checkData = async (request, h) => {
   // You can't hit this page directly, unless you've already uploaded a file
```

Comment:
I have not worked on it as it was there before. From my understanding, it is a page guard against users who have not uploaded files. But I will replace it by checking if the user has already uploaded files recorded on the database instead of checking if the user has uploaded files on the cache. As per your point, it might not guard when the session expires

Reply:
In general, when you are working on an application it needs to check if there is an application set, or redirect you to the /applications pages. 

See: packages/web-service/src/pages/common/check-application.js